libloki/api.js

```diff
@@ -107,11 +174,8 @@
     result.reset();
     return result;
   }
-  async function createContactSyncProtoMessage(conversations) {
+  async function createContactSyncProtoMessage(sessionContacts) {
```

Comment:
better to filter contacts before that call, so we only send chunk of 3 contacts (or less if last packet) @see https://github.com/loki-project/session-desktop/blob/clearnet/libtextsecure/sendmessage.js#L667-L672